common-app-lib/resources/public/site/cmr.css

```diff
@@ -0,0 +1,89 @@
+/* Home Pages
+ *
+ */
+
+.home.eui-hero {
+    background-size: cover;
+    background-color: #0d1f2b;
+    background-image: url('https://cdn.earthdata.nasa.gov/conduit/upload/947/LANCE_header_MODISAqua2.jpg');
+    background-repeat: no-repeat;
+}
+
+.home.description {
+  font-size: 150%;
+}
+
+/* Footer
+ *
+ * This section includes updates to other elements in order to give us a
+ * sticky footer.
+ */
+
+html {
+    height: 100%;
+}
+
+body {
+    min-height: 100%;
+    display: flex;
+    flex-direction: column;
+}
+
+main.content-section {
+    flex: 1;
+}
+
+footer {
+    background: #2276ac;
+    color: white;
+    padding: 1em 2em;
+    font-size: 0.8em;
+}
+
+footer a:link, footer a:visited {
+    color: white;
+}
+
+footer ul {
+    margin: 0.25em 0;
+    padding: 0;
+}
+
+footer ul li {
+    display: inline;
+}
+
+footer ul li:not(:first-child):before {
+    content: "\b7";
+    margin: 0 0.5em;
+}
+
+footer .badge {
```

Comment:
Lines 61 through 89 are not necessary if you are **not** going to have a badge component in your footer. If you look at EDSC or MMT you'll see a badge in the footer that denotes the version of the app you are looking at. It's handy for those apps but probably doesn't apply for this site.